main.js

```diff
@@ -0,0 +1,61 @@
+var btn=document.querySelector("#button");  //select button id
+
+btn.addEventListener('click',submitform);   //add click event to submit button
+
+function submitform(e){
+
+    e.preventDefault();
+
+    var inputtext=document.getElementById('input-item').value;  //select your input text
+    
+   // var li=document.createElement("li");
+    // Creating input field
+
+    var inputbox=document.createElement("input");
```

Comment:
use camelCase for variable naming as well, so ```inputbox``` should be``` inputBox```

Reply:
Done sir